Sources/Error Handling/ErrorUtils.swift

```diff
@@ -220,10 +230,10 @@ enum ErrorUtils {
         message: String? = nil,
         underlyingError: Error? = nil,
         fileName: String = #fileID, functionName: String = #function, line: UInt = #line
-    ) -> Error {
-        return error(with: ErrorCode.configurationError,
-                     message: message, underlyingError: underlyingError,
-                     fileName: fileName, functionName: functionName, line: line)
+    ) -> PurchasesError {
+        return self.error(with: ErrorCode.configurationError,
```

Comment:
Hmm there are a ton of these that call `error` without the `self`. Also, this is a static function so this shouldn't be needed I think? (I might be wrong)